apps/meteor/app/livechat/server/api/rest.js

```diff
@@ -10,3 +10,4 @@ import './v1/room.js';
 import './v1/videoCall.js';
 import './v1/transfer.js';
 import './v1/contact';
+import './v1/room.ts';
```

Comment:
How complex would be to convert the room.js file to ts?

Reply:
I tried to do that first however it was indeed getting complicated, hence I decided to just create a new ts file now for this particular endpoint & then we can migrate other endpoints one by one in this file on future eng days